docs/api.rst

```diff
@@ -27,6 +27,12 @@ Latest version of the header file:
 Python 3.11
 -----------
 
+.. c:function:: PyObject* PyCode_GetCode(PyObject *code)
```

Comment:
Nit: ``PyCodeObject *code`` not ``PyObject *code`` :)

Reply:
Fixed in commit bed155efcc133b0e901857fb4c964e84429087eb.